dipy/denoise/localpca.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import scipy as sp
+import scipy.linalg as sla
+from warnings import warn
+
+# Try to get the SVD through direct API to lapack:
+try:
+    from scipy.linalg.lapack import sgesvd as svd
+    svd_args = [1, 0]
+# If you have an older version of scipy, we fall back
+# on the standard scipy SVD API:
+except ImportError:
+    from scipy.linalg import svd
```

Comment:
That's correct. The direct interface to Lapack was introduced in Scipy 0.12, released in 2013.